File: java/src/main/java/euler/Problem007.java
package euler;

public class Problem007 {

    public long findNPrimeNumber(final int n) {
        long primeNumber = 1;
        long counterOfPrimeNumbers = 0;

        long currentNb = 1;
        while (counterOfPrimeNumbers < n) {
            currentNb++;

            long counter = 0;
            for (int i = 2; i <= currentNb; i++) {
                if (currentNb % i == 0) {
                    counter++;
                }
            }

            if (counter == 1) {
                counterOfPrimeNumbers++;
                primeNumber = currentNb;
            }
        }

        return primeNumber;
    }
}

File: python/src/euler/problem007.py
def find_n_prime_number(n: int) -> int:
    prime_number = 1
    counter_of_prime_numbers = 0

    current_nb = 1
    while counter_of_prime_numbers < n:
        current_nb += 1

        counter = 0
        for i in range(2, current_nb + 1):
            if current_nb % i == 0:
                counter += 1

        if counter == 1:
            counter_of_prime_numbers += 1
            prime_number = current_nb

    return prime_number